test.sh

```diff
@@ -0,0 +1,10 @@
+echo "dsdsd-$(echo 5)"
```

Comment:
if you intend to leave this file, please write instruction on how to use it in comments